lib/fitbank_api/version.rb

```diff
@@ -1,5 +1,5 @@
 # typed: strict
 module FitBankApi
-  VERSION = '0.2.0'
+  VERSION = '0.3.0'
```

Comment:
Should it not be a patch version change?

Reply:
I've added the base_url into the payout class. Technically this breaks the API of the function.